test/es-module/test-esm-detect-ambiguous.mjs

```diff
@@ -436,3 +436,20 @@ describe('when working with Worker threads', () => {
     strictEqual(signal, null);
   });
 });
+
+describe('Top-level await error in CommonJS modules', () => {
+  it('should throw a SyntaxError with guidance for top-level await in CommonJS', async () => {
+    const { code, stderr, stdout } = await spawnPromisified(process.execPath, [
+      '--eval',
+      'await Promise.resolve();',
+      '--input-type=commonjs',
```

Comment:
This file is `test-esm-detect-ambiguous.mjs`. The tests in this file are meant to concern ambiguous (rather than explicit) module types.

```suggestion
```